### ISSUE TITLE
Improve the build system to produce a multiplatform image

### ISSUE THREAD

**Issue by arcade-player** [ArcadeData/arcadedb#412]
## Expected behavior
Produce a multiplatform docker image using docker buildx

## Actual behavior
the build system produce a docker image just for intel arch

**Comment by kai-baumann** [comment]
Can I provide a PR to resolve this issue?

**Comment by tristanradams** [comment]
My team and I all use M1 macs and are blocked by this issue, is there any way we could bump up its priority?

**Comment by arcade-player** [comment]
Proposed solution: move to another base image (azul) that provides arm64/v8  image as well.
I'm going to open a PR in a couple of days.

**Comment by lvca** [comment]
Duplicate of #894

**Comment by lvca** [comment]
@robfrank with the latest commits, it doesn't compile on my Mac:

```
[ERROR] Errors:
[ERROR]   JdbcQueriesTest » NoClassDefFound Could not initialize class com.arcadedb.e2e.ArcadeContainerTemplate
[ERROR]   RemoteDatabaseQueriesTest.simpleCypherQuery » ExceptionInInitializer
[ERROR]   RemoteDatabaseQueriesTest.simpleGremlinQuery » NoClassDefFound Could not initialize class com.arcadedb.e2e.RemoteDatabaseQueriesTest
[ERROR]   RemoteDatabaseQueriesTest.simpleSQLQuery » NoClassDefFound Could not initialize class com.arcadedb.e2e.RemoteDatabaseQueriesTest
[INFO]
[ERROR] Tests run: 4, Failures: 0, Errors: 4, Skipped: 0
[INFO]
```

**Comment by robfrank** [comment]
I didn't touch the code . Only pom and GH workflows
